code/modules/capture_the_flag/ctf_equipment.dm

```diff
@@ -229,8 +219,7 @@
 	greyscale_config = /datum/greyscale_config/ctf_light
 	greyscale_config_worn = /datum/greyscale_config/ctf_light/worn
 	slowdown = -0.25
-
-	max_charges = 30
+	max_charges = 50
```

Comment:
👎 Assaulters are meant to be a high-risk high-reward playstyle, only being able to take one hit before losing your shield is a fair price for being able to do the same to other classes by shotgunning them in the face

Reply:
Are 2 hits instead of 1 to break shield makes such difference?